lib/virtual_device.js

```diff
@@ -121,6 +121,27 @@ VirtualDevice.prototype.call = function(/* transition, args, cb */) {
 
 };
 
+VirtualDevice.prototype.available = function(transition) {
+  return !!this._getAction(transition);
```

Comment:
Because the list of actions is eventually consistent, there's always a chance of this being out-of-sync.  Do we propagate a "Transition Not Available" error through virtual devices?